conf/application.conf

```diff
@@ -131,6 +131,7 @@ pluto {
 
 auth {
   tokenSigningCertPath = "keycloak-local.pem"
+  adminClaimName = "multimedia_admin"
```

Comment:
I'd put a placeholder in here for commit to the repo if i were you 😄

Reply:
Done.